sequencer/status.go

```diff
@@ -31,8 +31,12 @@ type Response struct {
 	Result Result `json:"result"`
 }
 
-func getRollappHeight(rollappID string) (string, error) {
-	resp, err := http.Get(fmt.Sprintf("%s/status", consts.DefaultRollappRPC))
+func getRollappHeight(rlpCfg config.RollappConfig) (string, error) {
+	rollappRPCEndpoint, err := GetRPCEndpoint(rlpCfg)
```

Comment:
IMO it should be improved,
no reason to read the port from the file **for each** status check

probably a Sequencer struct that holds the data on init will be a good solution (similar to what we have for relayer and da)